interface/gcp.py

```diff
@@ -86,35 +146,30 @@ def set_drive_permissions(self,
                     .create(fileId=drive_id,
                             body=body,
                             emailMessage=new_share_message(team_name),
-                            sendNotificationEmail=True,
-                            supportsAllDrives=True)\
+                            sendNotificationEmail=True)\
                     .execute()
                 created_shares += 1
             except Exception as e:
                 logging.error("Failed to share drive item"
                               + f"({team_name}, {drive_id}) with {email}: {e}")
         logging.info(f"Created {created_shares} permissions for {team_name}")
 
-        # Delete old permissions
+        # Delete unknown permissions
         # See http://googleapis.github.io/google-api-python-client/docs/dyn/drive_v3.permissions.html#delete # noqa
-        if delete_permissions is True:
-            deleted_shares = 0
-            for p_id in to_delete:
-                try:
-                    self.drive.permissions()\
-                        .delete(fileId=drive_id,
-                                permissionId=p_id,
-                                supportsAllDrives=True)\
-                        .execute()
-                    deleted_shares += 1
-                except Exception as e:
-                    logging.error(
-                        f'Failed to delete permission {p_id} for '
-                        + f'drive item ({team_name}, {drive_id}): {e}')
-            logging.info(
-                f'Deleted {deleted_shares} permissions for {team_name}')
-        else:
-            logging.info("DELETE_OLD_DRIVE_PERMISSIONS is set to false")
+        deleted_shares = 0
+        for p_id in to_delete:
+            try:
+                self.drive.permissions()\
+                    .delete(fileId=drive_id,
+                            permissionId=p_id)\
+                    .execute()
+                deleted_shares += 1
+            except Exception as e:
+                logging.error(
+                    f'Failed to delete permission {p_id} for '
+                    + f'drive item ({team_name}, {drive_id}): {e}')
+        logging.info(
+            f'Deleted {deleted_shares} permissions for {team_name}')
```

Comment:
Put into another function please.

Reply:
I'd rather keep this one here, since it is tightly coupled with the permissions creation - I don't want to expose an API that does anything other than `ensure` a permissions state (ie no create-only or delete-only APIs)